refactor: 📝 update the minutes of the in-person days (#366)
seedcase-project/team: events/2026-05-in-person/index.qmd

on building the products.

## Minutes

- We had some very good brainstorming and discussion time on Propagate's
  design. The details are in the issues in the repository, so I won't

>   repeat them here.

Reviewer: Link to them, maybe?

- We decided to be a lot more careful about choosing conferences as we
  didn't get as much out of it as we had hoped/anticipated. We have one
  more year of funding so we'll aim for the Research Software
  Engineering conference, which may be a better fit for us.

> - We spent a lot of time on the Propagate design so didn't get to do

Reviewer: ```suggestion
- We spent most of time on the Propagate design so we didn't get to do
```

>   nearly any outreach brainstorming, which is fine as it was a more

Reviewer: ```suggestion
  much brainstorming on outreach, which is fine as it was a more
```
